src/main/java/com/carpentersblocks/proxy/ClientProxy.java

```diff
@@ -11,13 +12,23 @@
 
 import cpw.mods.fml.client.FMLClientHandler;
 import cpw.mods.fml.client.registry.RenderingRegistry;
+import cpw.mods.fml.common.Loader;
 import cpw.mods.fml.common.event.FMLInitializationEvent;
+import cpw.mods.fml.common.event.FMLPreInitializationEvent;
 import cpw.mods.fml.relauncher.Side;
 import cpw.mods.fml.relauncher.SideOnly;
 
 @SideOnly(Side.CLIENT)
 public class ClientProxy extends CommonProxy {
 
+    @Override
+    public void preInit(FMLPreInitializationEvent event, Configuration config) {
+        super.preInit(event, config);
+        if (Loader.isModLoaded("Chisel")) {
+
+        }
+    }
```

Comment:
Is this piece of code a leftover or draft of your work? Please, remove this code if it's not needed.

Reply:
Was left over from prototyping.